docs/advanced/writing-mcps.md

```diff
@@ -0,0 +1,191 @@
+# Writing MCPs as Files
+
+## What is an MCP?
+
+A `MetadataChangeProposal` (MCP) represents an atomic unit of change in the DataHub Metadata Graph. Each MCP carries a single aspect in its payload and is used to propose changes to DataHub's metadata. 
+
+- Represents a single aspect change
+- Used for proposing metadata changes to DataHub
+- Serves as the basic building block for metadata ingestion
+
+For more information, please see [Guide in MCPs](https://datahubproject.io/docs/advanced/mcp-mcl/)
```

Comment:
we should also link to this https://datahubproject.io/docs/what/mxe

also, use relative links so that these don't break when we release new docs versions